### PR TITLE
Ios fallback fonts

### PR DESCRIPTION
**Pull request by karimnaaji**
Fixes https://github.com/tangrams/tangram-es/issues/643
Related https://github.com/tangrams/tangram-es/pull/649

### CODE REVIEW

**Comment by karimnaaji** [comment]
Merging this as is to fix issues we're having on iOS right now. Though, this would need to be cleaned up and to remove platform-dependent code/conditionals from core.

File: core/src/text/fontContext.cpp [hunk 1]
```diff
@@ -57,6 +59,36 @@ FontContext::FontContext() :
             m_font[i]->addFace(m_alfons.addFontFace(alfons::InputSource(fontPath), size));
         }
     }
+#elif defined(PLATFORM_IOS)
```
Comment (tallytalwar) [inline core/src/text/fontContext.cpp]: We definitely need a refactor of this once we have more knowledge of ios font loading, at least to get rid of the `PLATFORM_IOS` specific handling here.

**Comment by MajidBashir** [comment]
well related to fonts, sometimes application is crashing over fonts

> NOTIFY textStyleBuilder.cpp:300: stroke_width too large: 12.094118 / 2.015686


how can i fix this issue

**Comment by blair1618** [comment]
@MajidBashir this crash doesn't seem to be related to fonts - that "NOTIFY" message is only a warning. If you are still having this problem could you please open a new issue?

**Pull request merged**